src/_main_/settings.py

```diff
@@ -63,6 +70,7 @@
     'massenergize-canary-api.us-east-2.elasticbeanstalk.com',
     'massenergize.test',
     'massenergize.test:3000',
+    'communities.massenergize.test'
```

Comment:
massenergize.test covers this OR you could also do *.massenergize.test

Reply:
ok